interfaces/builtin/basedeclaration.go

```diff
@@ -195,9 +195,28 @@ slots:
       slot-snap-type:
         - app
         - gadget
+    allow-connection:
+      -
+        plug-attributes:
+          content: $SLOT(content)
+      -
+        plug-attributes:
+          content: $MISSING
+        slot-attributes:
+          content: $MISSING
```

Comment:
Hmm.. that doesn't seem right. What would be a content interface with a missing "content" field?  I thought the $MISSING case was for a future need.